Week3/homework/js-exercises/4_GroceryShopping.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+    
+function addToShoppingCart(groceryItem) {
+    
+    if (cart.length > 2) {
+        cart.push(groceryItem);
+        cart.shift();
+    } else {
+        cart.push(groceryItem);
+    }
```

Comment:
I think you can simplify this logic like:
```
cart.push(item);
if (cart.length > 2) {
  cart.shift(); 
}
```

What do you think? 🙂

Reply:
Much better. But I guess in that case I need to make: if (cart.length > 3).